environment/MetaData.go

```diff
@@ -0,0 +1,96 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I think that this comment got missed somewhere. The file name shouldn't be capitalized.